packages/ui-extensions/src/surfaces/admin/components/CustomerSegmentTemplate/CustomerSegmentTemplate.doc.ts

```diff
@@ -0,0 +1,39 @@
+import {ReferenceEntityTemplateSchema} from '@shopify/generate-docs';
+
+const data: ReferenceEntityTemplateSchema = {
+  name: 'CustomerSegmentTemplate',
+  description:
+    'CustomerSegmentTemplate is used to configure a template used in the customers section in the admin.',
+  requires: 'use of [admin.customers.segmentation-templates.render](/docs/api/admin-extensions/api/extension-targets#extensiontargets-propertydetail-admincustomerssegmentationtemplatesrender) target.',
+  isVisualComponent: true,
+  type: '',
+  definitions: [
+    {
+      title: 'CustomerSegmentTemplateProps',
```

Comment:
this is consistent throughout this set of docs, but this should really be written as a sentence: "CustomerSegmentTemplate props"